### ISSUE TITLE
Journal today, Journal insert today note link, go to another day

### ISSUE THREAD

**Issue by juliux-arg** [leenzhu/joplin-plugin-journal#8]
Today is Oct 24. However, Journal today, and Journal insert today both go to Oct 8. Journal version is 1.05 (just updated version)

**Comment by leenzhu** [comment]
It is really strange. Could you please provide the output of `new Date()` 
1. Start joplin
2. Click menu `Help`->`Toggle Development tools`
3. Click `Console` tab
4. type `new Date()`
5. Copy the output here.

Here is mine:

```
new Date()
Tue Oct 25 2022 09:17:15 GMT+0800 (中国标准时间)
```

**Comment by juliux-arg** [comment]
This is the output:

```
new Date()
Mon Oct 24 2022 22:22:07 GMT-0300 (hora estándar de Argentina)
```

**Comment by leenzhu** [comment]
I've changed my time zone as yours, it works as expected, I'm not sure what happend to your joplin.

Please try the following code in console:

```js
const d = new Date()
const ds = new Date(`${d.getFullYear()}-${d.getMonth() + 1}-${d.getDate()}`);
ds.toUTCString()
```

it shuold output:

```
'Mon, 24 Oct 2022 00:00:00 GMT'
```

**Comment by juliux-arg** [comment]
Now it is Oct 25 17:29. The output in console is:

`'Tue, 25 Oct 2022 00:00:00 GMT'`

**Comment by juliux-arg** [comment]
¡It's solved! I moved offending note 08 to another notebook out of Journal. Then everything works correctly.
Then I moved again note 08 to Journal/2022/10, and it's OK now.

**Comment by leenzhu** [comment]
glade to hear that.&nbsp;

---Original---
From: ***@***.***&gt;
Date: Wed, Oct 26, 2022 04:56 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [leenzhu/joplin-plugin-journal] Journal today, Journal inserttoday note link, go to another day (Issue #8)




 
Closed #8 as completed.

**Comment by leenzhu** [comment]
I'm afraid I have to reopen this issue.

After reviewing the code I found that is the note's content contains note tile, the search will fail.
